lib/core/pktmbuf/pktmbuf.c

```diff
@@ -104,38 +107,69 @@ pktmbuf_iterate(pktmbuf_info_t *pi, pktmbuf_cb_t cb, void *ud)
     return 0;
 }
 
+int
+pktmbuf_pool_cfg(pktmbuf_pool_cfg_t *c, void *addr, uint32_t bufcnt, uint32_t bufsz,
+                 uint32_t cache_sz, void *metadata, uint32_t metadata_bufsz, mbuf_ops_t *ops)
+{
+    if (!c || !addr || !bufcnt || !bufsz)
```

Comment:
This needs to change to allow `bufcnt` and `bufsz` to be 0.

You might as well do the same thing that is done in pktmbuf_pool_cfg_create()...

```
    CNE_MAX_SET(cache_sz, MEMPOOL_CACHE_MAX_SIZE);
    CNE_DEFAULT_SET(cache_sz, 0, MEMPOOL_CACHE_MAX_SIZE);
    CNE_DEFAULT_SET(bufcnt, 0, DEFAULT_MBUF_COUNT);
    CNE_DEFAULT_SET(bufsz, 0, DEFAULT_MBUF_SIZE);

    if (CNE_CACHE_LINE_ROUNDUP(bufsz) != bufsz)
        err!!!
```